Sources/main.cpp

```diff
@@ -399,13 +217,12 @@ int main(int argc, char **argv) {
     env.erase(e);
   }
 
-  if (!opts.getString("attach").empty()) {
-    attachPid = atoi(opts.getString("attach").c_str());
-  }
+  int attachPid = opts.getString("attach").empty()
+                      ? -1
+                      : atoi(opts.getString("attach").c_str());
```

Comment:
::atoi?

Reply:
There are a bunch of these from before I touched this code. I'll change all of them at once in a separate commit.